main/main.pde

```diff
@@ -41,101 +40,102 @@ void setup()
   kinect.initDepth();
   kinect.enableMirror(true);
   background(0);
-  particleSystem.addParticle(new PVector(random(width), random(height)));
+  // particleSystem.addParticle(new PVector(random(width), random(height)));
```

Comment:
Do we still need this?

Reply:
Right! We don't, ill get rid of it.

Reply:
Removed here: 6aec219b063e48ed8b6752364a7a71583154636b